pose/README.md

```diff
@@ -201,15 +201,13 @@ This method exists on the model that is loaded from `tmPose.load`.
 ```ts
 model.predict(
     poseOutput: Float32Array,
```

Comment:
why was `flipped` taken off of `predict` ? This makes the method signatures between pose and image different which is confusing.

Reply:
Just logging here we talked about it and are going to go with 1) function name changes and 2) moving the flag as last variable.

Reply:
Now the functions are `predict` and `predictTopK`